src/gatsby/models/blog/onCreateMarkdownContentNode.js

```diff
@@ -0,0 +1,52 @@
+const { markdownToHtml } = require('../../common.js')
+
+function createMarkdownBlogNode(api, { parentNode }) {
+  if (parentNode.relativeDirectory.split('/')[0] !== 'blog') return
+  const { node, actions, createNodeId, createContentDigest } = api
+  const { createNode, createParentChildLink } = actions
+  const { frontmatter, rawMarkdownBody } = node
+  const {
+    date,
+    tags,
+    title,
+    author,
+    description,
+    descriptionLong,
+    commentsUrl,
+    picture,
+    pictureComment
+  } = frontmatter
+  const { name, relativePath } = parentNode
+
+  const slug = name === 'index' ? '' : /[-\d]*(.*)/.exec(name)[1]
```

Comment:
if we don't need it - let's remove?

Reply:
Ah yes, I meant to remove this. I put it down out of habit before I realized no blog entries are named `index`.